matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/homeserver/GetHomeServerCapabilitiesTask.kt

```diff
@@ -134,6 +135,7 @@ internal class DefaultGetHomeServerCapabilitiesTask @Inject constructor(
                 }
                 homeServerCapabilitiesEntity.canUseThreading = /* capabilities?.threads?.enabled.orFalse() || */
                         getVersionResult?.doesServerSupportThreads().orFalse()
+                homeServerCapabilitiesEntity.canLoginWithQrCode = getVersionResult?.doesServerSupportQrCodeLogin().orFalse()
```

Comment:
Should this assignment and the one about `canUseThreading` be moved into the `if(getVersionResult != null)` for consistency? Here we are not relying on the `getCapabilitiesResult`.

Reply:
Good catch, done.